kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Filters/Filters.tsx

```diff
@@ -344,11 +340,11 @@ const Filters: React.FC<FiltersProps> = ({
   ]);
   React.useEffect(() => {
     if (location.search?.length === 0) {
-      handleFiltersSubmit();
+      handleFiltersSubmit(offset);
```

Comment:
shouldn't offset be in the dependency array otherwise it will get some outdated variable ?

Reply:
No. Offset is an input value state. If I added offset to the dependencies, the form would be submitted every time the input changed.